app/src/main/java/org/oppia/app/player/state/hintsandsolution/HintsAndSolutionFragmentPresenter.kt

```diff
@@ -95,4 +133,10 @@ class HintsAndSolutionFragmentPresenter @Inject constructor(
   private fun handleNewAvailableHint(hintIndex: Int) {
     hintsAndSolutionAdapter.setNewHintIsAvailable(hintIndex)
   }
+
+  fun onExpandClicked(index: Int?) {
+    currentExpandedHintListIndex = index
+    if(index!=null)
+    hintsAndSolutionAdapter.notifyItemChanged(index)
```

Comment:
Formatting can be fixed here.
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done.